### DISCUSSION TITLE
preventing a rule from autogenerating

### DISCUSSION THREAD

**Discussion by 96312** [tailwindlabs/tailwindcss#11801] [Help] [answered]
Please tell me a way to prevent the tailwindcss build process from automatically including this set of rules in my final CSS file. I don't want it.
```
ol,
ul,
menu {
  list-style: none;
  margin: 0;
  padding: 0;
}
```

**Comment by wongjn** [comment, marked as answer]
You could consider:
 - [Disabling preflight](https://tailwindcss.com/docs/preflight#disabling-preflight) Tailwind core plugin (and running your own version if required).
 - [Having a PostCSS plugin](https://postcss.org/docs/writing-a-postcss-plugin) that runs after Tailwind that looks for that specific rule and removes it from the CSS.

**Reply by 96312** [reply]
Does preflight disabling mess with other rules that are autogenerated?

**Reply by wongjn** [reply]
Disabling preflight can cause other CSS rules to not function properly.

**Reply by 96312** [reply]
Then that is not what I will do. Maybe the 2nd option is better. 
On one page in my application, I need these default styles, in the other, I don't need list-style, list margins and list paddings. 
Thanks for your suggestion.
